app/controllers/application_controller.rb

```diff
@@ -26,6 +28,19 @@ class ApplicationController < ActionController::Base
 
 private
 
+  def skip_session_cookie
+    unless current_user
+      request.session_options[:drop] = true
+    end
+  end
+
+  def logout!
+    session.delete(:sub)
+    session.delete(:email)
```

Comment:
This `session.delete(:email)` is no longer needed

Reply:
Missed that one!   Have removed.